src/main/java/Duke.java

```diff
@@ -1,10 +1,158 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String COMMAND_LIST_WORD = "list";
+    private static final String COMMAND_TODO_WORD = "todo";
+    private static final String COMMAND_DEADLINE_WORD = "deadline";
+    private static final String COMMAND_EVENT_WORD = "event";
+    private static final String COMMAND_DONE_WORD = "done";
+    private static final String COMMAND_EXIT_WORD = "bye";
+
+    private static final String LOGO = "\t   .            *        .\n" +
+                                        "\t *    .     * .     *\n" +
+                                        "\t  .         ___  .        *\n" +
+                                        "\t   *      _[___]_   .\n" +
+                                        "\t *  .      ('▻') v *    *\n" +
+                                        "\t        >--( . )/     .     .\n" +
+                                        "\t .    *   (  :  )   *\n" +
+                                        "\t .. . ...  '--`-` ... *  .";
+
+
+    private static final Task[] list = new Task[100];
```

Comment:
It will be better to set a constant like MAX_TASK_LENGTH = 100 as to avoid the use of magic numbers

Reply:
I have edited this, thank you for the advice!